api/requirements-dev.txt

```diff
@@ -0,0 +1,5 @@
+coverage~=7.2.4
+pytest~=7.3.1
+pytest-mock~=3.11.1
+pytest-benchmark~=4.0.0
+ruff~=0.3.5
```

Comment:
```suggestion
ruff~=0.3.5
-r requirements.txt
```

or on the top

Reply:
Hi, the `requirements.txt` is for production dependencies, which is not included in `requirements-dev.txt` on purpose by design. I'm trying to keep `requirements-dev.txt` small and available for testing, lining and other development purposes only.